webapp/components/experiments/ExperimentEdit.js

```diff
@@ -512,43 +529,53 @@ const ExperimentEdit = (props) => {
                   </Box>
                 )}
               </Box>
-            )}
-            {selectionParams.length > 0 && (
+              )}
+              {selectionParams.length > 0 && (
               <Box className="editExperimentRow">
                 {parameters.map((parameter, index) => (
                   ParameterRow(parameter, index, handleParamSelection, handleChange,
                     handleRangeInput, handleInputValues, addToGroup, removeFromGroup,
                     removeParameter, selectionParams, classes)
                 ))}
               </Box>
-            )}
+              )}
+            </Box>
+          </Box>
+          <Box>
+            <Link to="true" color="primary" onClick={addParameter}>
+              <AddIcon />
+              Add parameter
+            </Link>
           </Box>
         </Box>
-        <Box>
-          <Link to="true" color="primary" onClick={addParameter}>
-            <AddIcon />
-            Add parameter
-          </Link>
-        </Box>
-      </Box>
-      <Box
-        className="scrollbar scrollchild"
-        mt={1}
-        display="flex"
-        flexWrap="wrap"
-      >
-        <Box className="editExperimentFooter">
-          <Box display="flex">
-            <Button color="secondary" onClick={() => setView(EXPERIMENT_VIEWS.list)}>
-              Cancel
-            </Button>
-            <Button variant="contained" color="primary" onClick={submit}>
-              {editState ? 'Save' : 'Create'}
-            </Button>
+        <Box
+          className="scrollbar scrollchild"
+          mt={1}
+          display="flex"
+          flexWrap="wrap"
+        >
+          <Box className="editExperimentFooter">
+            <Box display="flex">
+              <Button color="secondary" onClick={() => setView(EXPERIMENT_VIEWS.list)}>
+                Cancel
+              </Button>
+              <Button variant="contained" color="primary" onClick={submit}>
+                {editState ? 'Save' : 'Create'}
+              </Button>
+            </Box>
           </Box>
         </Box>
-      </Box>
-    </GridLayout>
+      </GridLayout>
+      <DialogBox
+        open={trialNumberErrorDialogOpen}
+        onDialogResponse={() => setTrialNumberErrorDialogOpen(false)}
+        textForDialog={{
+          heading: 'Error - Number of trials is too large',
```

Comment:
Were not trials renamed to "combinations"? cc @ddelpiano

Reply:
It also says 'trials' when viewing an experiment's details. I can fix the wording here too once we know what word we're using.

![image](https://user-images.githubusercontent.com/40673427/143280961-bf0aa26b-9e66-4b54-b49b-926941c98556.png)

Reply:
I'm afraid there is some misalignment here, let's see what @ddelpiano and @enicolasgomez remember about this

Reply:
it's a leftover from [this](https://app.zenhub.com/workspaces/netpyne---metacell-board-5bd615394b5806bc2bef3799/issues/metacell/netpyne-ui/368), my bad I missed @enicolasgomez comments down in the card, I will look at that and this in the afternoon today.